README.md

```diff
@@ -1296,6 +1296,45 @@ _You can enable the following settings in Xcode by running [this script](resourc
   
   </details>
 
+* <a id='extension-access-control'></a>(<a href='#extension-access-control'>link</a>) **Specify the access control for each declaration in an extension individually.** [![SwiftFormat: extensionAccessControl](https://img.shields.io/badge/SwiftFormat-anyObjectProtocol-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md##extensionaccesscontrol)
+
+  <details>
+
+  #### Why?
+
+  Specifying the access control on the declaration itself helps engineers more quickly determine the access control level of an individual declaration.
+
+  ```swift
+  // WRONG
+  public extension Foo {
```

Comment:
Totally optional: what do you think of coming up with more fun names?

Reply:
Sure, I could use the `Spaceship` example that we use elsewhere in the style guide.

Reply:
Updated: 

  ```swift
  extension Universe {
    public func generateGalaxy() { }
  }

  extension Spaceship {
    fileprivate func enableHyperdrive() { }
  }
  ```